lib/kudos.ex

```diff
@@ -31,9 +31,19 @@ defmodule Kudos do
     """
   end
 
+  defp format(:umbrella) do
+    ""
+  end
+
+  defp format(:dev) do
+    ""
+  end
+
   defp format(meta_data) do
     """
-    ### #{meta_data.name} (Version #{meta_data.version} | Checksum: #{checksum(meta_data.checksum)})
+    ### #{meta_data.name} (Version #{meta_data.version} | Checksum: #{
+      checksum(meta_data.checksum)
+    })
```

Comment:
```suggestion
    ### #{meta_data.name} (Version #{meta_data.version} | Checksum: #{checksum(meta_data.checksum)})
```
Must be formatted like that otherwise the elixir 12 formatter will complain.